modules/mobile/docs/openapi.yaml

```diff
@@ -2435,6 +2435,74 @@ paths:
       security:
         - Bearer: []
       summary: /v0/messaging/health/folders/{id}/messages
+  /v0/messaging/health/folders/{id}/threads:
+    get:
+      description: List of threads in a secure messaging folder
+      operationId: get_folder_threads
+      parameters:
+        - description: The id of the folder that threads are being retrieved from
+          in: path
+          name: folderId
+          required: true
+          schema:
+            type: string
+        - description: The size of the pagination you want. Defaults to 10
+          in: query
+          name: pageSize,
+          schema:
+            type: string
+        - description: The page number to get based on your page size
+          in: query
+          name: page,
+          schema:
+            type: string
+        - description: The field to sort the results by
+          in: query
+          name: sortField,
+          schema:
+            type: string
+            enum: [ SENDER_NAME, RECIPIENT_NAME, SENT_DATE, DRAFT_DATE ]
+        - description: The order to sort the results by
```

Comment:
Do you know if this defaults to ASC or DESC? Could be worth capturing that and also the fact that page defaults to 1 (even though that's pretty obvious).

Reply:
DESC I believe. I just tried sender_name and the order was `ISLAM, MOHAMMAD  RAFIQ` then `GOPARAJU, BHANU `